DESCRIPTION

```diff
@@ -8,7 +8,9 @@ Description: Provides an Rmd macro to insert
 Depends:
     R (>= 3.3.0)
 Imports:
-    gtools
+    gtools,
+    RCurl,
+    utils
```

Comment:
You're welcome. I wasn't necessarily aiming to make the package CRAN-ready (there were several other warnings in `devtools::check()` that I didn't address), but I agree that it's a good goal to aspire to, even it the package never makes it on CRAN. Let me know if there is anything else you'd like me to do on this PR.